.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  # Pub dependencies
+  - package-ecosystem: "pub"
+    directories:
+      - packages/**/*
+      - modules/**/*
+      - integrations/**/*
+      - templates/serverpod_templates
+      - tools/serverpod_cli
+    schedule:
+      interval: "daily"
+    versioning-strategy: "widen"
+    # Since we are using a monorepo, we want to update the same package
+    # in all directories at the same time.
+    # Currently dependabot does not support this, so we limit the number of
+    # open pull requests to 1 to avoid conflicts.
```

Comment:
Question: Have you ever discussed what tone of voice you'd like to have in in-code documentation/comments? This one is for Serverpod developers only, but since it's open source it's kind of "public documentation" as well. Perhaps it should be that same as in https://docs.serverpod.dev/? In that case the comment would avoid words like "we" and be more formal
```
Since this is a monorepo, all packages in all directories should be updated at the same time.
Etc...
```